modules/oe_whitelabel_search/src/Plugin/Block/SearchBlock.php

```diff
@@ -62,12 +69,15 @@ class SearchBlock extends BlockBase implements ContainerFactoryPluginInterface {
    *   The form builder service.
    * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
    *   The module handler.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager.
    */
-  public function __construct(array $configuration, $plugin_id, $plugin_definition, ConfigFactoryInterface $config_factory, FormBuilderInterface $form_builder, ModuleHandlerInterface $module_handler) {
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, ConfigFactoryInterface $config_factory, FormBuilderInterface $form_builder, ModuleHandlerInterface $module_handler, EntityTypeManagerInterface $entity_type_manager) {
     parent::__construct($configuration, $plugin_id, $plugin_definition);
     $this->configFactory = $config_factory;
     $this->formBuilder = $form_builder;
```

Comment:
Please note that formBuilder and configFactory are two services not use in the block. But I've kept them in case anyone extended this class.